### PR TITLE
S4ii

### PR DESCRIPTION
**Pull request by imypullan**
all good for peer review. everything up to date.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        return true;
+    } else {
+        return false;
+    }
+}
+
+startBtn.addEventListener("click", () => {
+    startBtn.style.opacity = 0.5;
+    tiles.forEach(function (tile) {
+        tile.addEventListener('click', () => {
```
Comment (dotNATE) [inline script.js:40]: Does this get removed? i.e. If I press start game again will there be 2 event listeners on this square

Comment (imypullan) [reply script.js:40]: this is fixed

File: script.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    }
+}
+
+function playGame() {
+    // this determines turn order
+    let currentPlayer = 1;
+    // this variable holds alternating player symbols
+    let playerMarker;
+    const tiles = document.querySelectorAll('.tile');
+    let gameState = ["", "", "", "", "", "", "", "", ""];
```
Comment (dotNATE) [inline script.js]: Can this be made into a global variable please?

Comment (dotNATE) [reply script.js]: So that we can access when deciding victory

**Pull request closed without merging**